README.md

```diff
@@ -1,40 +1,51 @@
-# Black Magic Probe
-
-Firmware for the Black Magic Debug Probe.
-
-The Black Magic Probe is a modern, in-application debugging tool for
-embedded microprocessors. It allows you see what is going on 'inside' an
-application running on an embedded microprocessor while it executes. It is
-able to control and examine the state of the target microprocessor using a
-JTAG or Serial Wire Debugging (SWD) port and on-chip debug logic provided
-by the microprocessor. The probe connects to a host computer using a
-standard USB interface. The user is able to control exactly what happens
-using the GNU source level debugging software, GDB.
-Serial Wire Output (SWO) allows the target to write tracing and logging to the host
-without using usb or serial port. Decoding SWO in the probe itself
-makes [SWO viewing as simple as connecting to a serial port](https://black-magic.org/usage/swo.html).
+# Black Magic Debug
+
+The Black Magic Debug suite is a self-contained debugger for microcontrollers which needs no config
+files and auto-detects plus configures the connected targets. It is designed to be fast and easy to use.
+
+The project can be built as either firmware for [Black Magic Probe](https://1bitsquared.com/products/black-magic-probe) -
+a debugger-in-a-dongle that provides multi-voltage debug with no external tools than GDB required -
```

Comment:
> with no external tools than GDB required

This doesn't sound quite right, is a word missing?

Reply:
Good call - you are indeed correct :)